youtube_dl/extractor/kaltura.py

```diff
@@ -135,7 +140,7 @@ def _extract_url(webpage):
                 ''', webpage)
             or re.search(
                 r'''(?xs)
-                    <(?:iframe[^>]+src|meta[^>]+\bcontent)=(?P<q1>["'])
+                    <(?:iframe[^>]+src|meta[^>]+\bcontent)=(?P<q1>["'])[ ]?
```

Comment:
Superfluous brackets. Doesn't cover multiple spaces. Doesn't cover other kinds of whitespace.

Reply:
Brackets are necessary because it has the X option set, which makes it ignore whitespace outside a character class.

I'll augment it to cover unlimited amounts of all whitespace, however.